components/tokens/custom-spectrum/custom-light-vars.css

```diff
@@ -31,4 +31,5 @@ governing permissions and limitations under the License.
   --spectrum-calendar-day-border-color-key-focus: var(--spectrum-blue-800);
 
   --spectrum-badge-label-icon-color-primary: var(--spectrum-white);
+  --spectrum-well-border-color:rgba(var(--spectrum-black-rgb), 0.05);
```

Comment:
Looks like we only need the opacity on the border color for express. you can handle this by adding these values to `components/tokens/custom-express/custom-light-vars.css` and `components/tokens/custom-express/custom-dark-vars.css` with the opacity and removing the opacity from where it is here and in `components/tokens/custom-spectrum/custom-dark-vars.css`